### DISCUSSION TITLE
PromQL Builder Support no-ascii metric name and label

### DISCUSSION THREAD

**Discussion by llm-infer** [grafana/grafana#60206] [Dashboards, Panels & Visualizations]
### The feature I prompt
PromQL Builder can add escaping char '\\' before each no-ascii char when generating the PromQL.

### Why is this needed?
There are some Prometheus-compatible datasource like [VictoriaMetrics](https://victoriametrics.com/) already support no-ascii metric name and label. The valid query is like` \€_price{temperature_\°C="123"}`. But in PromQL Builder, I get `€_price{temperature_°C="123"}` which is not unrecognizable.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
